src/test/java/bio/terra/common/TestUtils.java

```diff
@@ -230,24 +230,44 @@ public static TableResult selectFromBigQueryDataset(
   }
 
   public static <T> T mapFromJson(String content, Class<T> valueType) throws IOException {
+    if (content == null) {
+      return null;
+    }
     try {
       return objectMapper.readValue(content, valueType);
-    } catch (IOException ex) {
+    } catch (Exception ex) {
```

Comment:
IMO we should remove these methods that wrap `readVaule()` and instead let the exceptions be thrown out of our methods. These exceptions will fail the test, which is what we want. Logging here is unnecessary as the top level catch will log the exception and print a stack crawl.